apps/contributions/models.py

```diff
@@ -343,5 +345,13 @@ def handle_thank_you_email(self, contribution_received_at=None):
                     "contribution_interval_display_value": self.interval if self.interval != "one_time" else None,
                     "copyright_year": contribution_received_at.year,
                     "org_name": self.revenue_program.organization.name,
+                    # TODO: Missing field to be added on DEV-2892
+                    # "first_name": '',
+                    # "last_name": '',
+                    # "for_profit": '',
```

Comment:
See comments below. This is better as original field name from RP.

```suggestion
                    # "non_profit": '',
```